package ru.practicum.shareit.user.exceptions;

public class UpdateUserException extends RuntimeException {
  public UpdateUserException(Throwable cause) {
    super(cause);
  }
}
